tool/bazelinstall/rbe.sh

```diff
@@ -31,13 +31,10 @@ function install_dependencies() {
     fi
 }
 
-cd /opt/circleci/.pyenv/plugins/python-build/../.. && git pull && cd -
-
 if [[ -n "$BAZEL_BUILDBUDDY_CERT" && -n "$BAZEL_BUILDBUDDY_KEY" ]]; then
     echo "Installing BuildBuddy credential..."
-    BAZEL_BUILDBUDDY_CREDENTIAL=/home/circleci/.credentials/
+    BAZEL_BUILDBUDDY_CREDENTIAL=/opt/.credentials/
     echo "A BuildBuddy credential is found and will be saved to $BAZEL_BUILDBUDDY_CREDENTIAL. Targets will be built and tested remotely."
-    mkdir -p $BAZEL_BUILDBUDDY_CREDENTIAL
```

Comment:
Does this directory always exist (`/opt/.credentials`)?

Reply:
@lriuui0x0 it will, in a newly-generated Azure Grabl image

Reply:
@lriuui0x0 addressed in https://github.com/graknlabs/dependencies/pull/176/commits/41f6b82f1b3c9ce2d3007aca9ef5a92a12c30a0c